rust/lance/src/io/exec/take.rs

```diff
@@ -488,19 +497,22 @@ mod tests {
     async fn test_with_new_children() -> Result<()> {
         let dataset = create_dataset().await;
 
+        let config = LanceScanConfig {
+            batch_size: 10,
+            batch_readahead: 10,
+            fragment_readahead: Some(4),
+            io_buffer_size: *DEFAULT_IO_BUFFER_SIZE,
+            with_row_id: true,
+            with_row_address: false,
+            with_make_deletions_null: false,
+            ordered_output: true,
+        };
```

Comment:
To improve clarity of test, can only set the parameters that are important:
```suggestion
        let config = LanceScanConfig {
            with_row_id: true,
            ..Default::default(),
        };
```